pkg/apis/apis.go

```diff
@@ -18,16 +18,18 @@ import (
 	_ "embed"
 
 	"github.com/samber/lo"
-	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
+	v1 "sigs.k8s.io/karpenter/pkg/apis/v1"
```

Comment:
Do you have to alias this? 
```suggestion
	"sigs.k8s.io/karpenter/pkg/apis/v1"
```